handlers/kannel/kannel_test.go

```diff
@@ -29,7 +29,7 @@ var handleTestCases = []ChannelHandleTestCase{
 	{Label: "Receive Empty Message", URL: receiveEmptyMessage, Data: "empty", Status: 200, Response: "Accepted",
 		Text: Sp(""), URN: Sp("tel:+2349067554729"), ExternalID: Sp("asdf-asdf"), Date: Tp(time.Date(2017, 5, 2, 14, 31, 49, 0, time.UTC))},
 	{Label: "Receive No Params", URL: receiveNoParams, Data: "empty", Status: 400, Response: "field 'sender' required"},
-	{Label: "Invalid URN", URL: receiveInvalidURN, Status: 400, Response: "phone number supplied was empty"},
+	{Label: "Invalid URN", URL: receiveInvalidURN, Data: "empty", Status: 400, Response: "phone number supplied was empty"},
```

Comment:
all this time I kept looking in the wrong place I was thing the error was in junebug